### PR TITLE
Add a manual entry for deploying the service unavailable page.

### PR DESCRIPTION
**Pull request by MancunianSam**
There are also instructions on how to restore the TDR app.

### CODE REVIEW

File: manual/README.md [hunk 1]
```diff
@@ -42,6 +42,10 @@ things like deploying code or debugging complex parts of the stack.
 * [Find S3 upload folder](debugging/find-s3-upload-folder.md)
 * [Security log queries (Sharepoint)](https://nationalarchivesuk.sharepoint.com/:w:/r/sites/DA_Proj/Transfer%20Digital%20Records/Documentation/Developer%20manual/Security%20log%20queries.docx?d=w757b4033c718479eb0b1050a8328eb46&csf=1&web=1&e=fh1pOE)
 
+### Deploy the service unavailable page
+
+* [Deploy the service unavailable page](deploy-the-service-unavailable-page.md)
```
Comment (TomJKing) [inline manual/README.md:47]: Maybe put this under the `Admin Tasks` heading?

**Review by TomJKing — COMMENTED** [review]
This looks good to me. Just one minor suggestion, feel free to ignore.

File: manual/README.md (hunk 1 above)
Comment (MancunianSam) [reply manual/README.md]: That makes sense. I've moved the link.

**Review by TomJKing — APPROVED** [review]
Looks good.

**Pull request merged**